.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+Thanks for taking the time for reporting an issue!
+
+Before reporting an issue on dd-trace-py, please be sure to provide all
+necessary information.
```

Comment:
Can we be more specific about what "all necessary information" is? For bugs we could probably create a list of things to provide (library versions - maybe `requirements.txt` or `Pipfile`, python version, etc).

Reply:
Good point, I'll add a question about `pip freeze`

Reply:
Well, below he asks specific questions.

I thought about asking for `requirements.txt`, but not sure it is entirely necessary always?

Reply:
How about we roll these into a `ddtrace-run doctor`?

Reply:
well, when we have it, sure :)